src/Reflection/SignatureMap/functionMap.php

```diff
@@ -2022,6 +2022,10 @@
 'DOTNET::__construct' => ['void', 'assembly_name'=>'string', 'class_name'=>'string', 'codepage='=>'int'],
 'dotnet_load' => ['int', 'assembly_name'=>'string', 'datatype_name='=>'string', 'codepage='=>'int'],
 'doubleval' => ['float', 'var'=>'mixed'],
+'Ds\\Map::first' => ['Ds\\Pair'],
```

Comment:
Don't double the backslashes. Other similar rows in this file do not double them.